tests/test_nodes_no_host.py

```diff
@@ -52,3 +52,34 @@ def test_algorithm():
     assert a.type == "integration"
 
     #Add citation test, once we have citation implemted
+
+def test_search():
```

Comment:
I recommend docstrings for this test to explains its purpose and what it tries to test for

Reply:
done